poetry/installation/pip_installer.py

```diff
@@ -232,18 +236,21 @@ def install_directory(self, package):
 
                 with builder.setup_py():
                     if package.develop:
-                        args.append("-e")
-
+                        return pip_editable_install(req, self._env)
                     args.append(req)
 
                     return self.run(*args)
 
         if package.develop:
-            args.append("-e")
+            return pip_editable_install(req, self._env)
 
         args.append(req)
 
-        return self.run(*args)
+        with temporary_directory() as tmp_dir:
```

Comment:
I think this could use `pip_install` from the helpers, no?

Reply:
Correct. Fixed.